src/components/displaySettings/displaySettings.template.html

```diff
@@ -63,50 +78,65 @@ <h2 class="sectionTitle">
     <div class="selectContainer fldDateTimeLocale hide">
         <select is="emby-select" class="selectDateTimeLocale" label="${LabelDateTimeLocale}">
             <option value="">${Auto}</option>
+            <option value="af">Afrikaans</option>
+            <option value="sq">Albanian</option>
             <option value="ar">Arabic</option>
-            <option value="be-BY">Belarusian (Belarus)</option>
-            <option value="bg-BG">Bulgarian (Bulgaria)</option>
+            <option value="be-BY">Belarusian</option>
+            <option value="bn_BD">Bengali (Bangladesh)</option>
```

Comment:
Some of these don't exist as date-fns locales. Should they all be?

Also some of them don't match the name of the locales in dfnshelper. There seems like there may be another place where they get normalized as some of the existing locales also don't match, but I'm not familiar with where or how that happens.

Reply:
I was hoping you knew lol. I noticed it was being parsed differently so I wasn't sure how to verify my work.

Reply:
Our locales are here: https://github.com/jellyfin/jellyfin-web/blob/master/src/scripts/site.js#L71

The strings in dfnshelper matches those locales using our code as a key with the date fns locale object as value

Reply:
mr wasnt a locale listed so i removed it, it should be good after that.